### DISCUSSION TITLE
Can  anyone help me to run this on my system ?

### DISCUSSION THREAD

**Discussion by Wajid4040** [jatinarora1/Theft-Detection#11] [Q&A]
Hi dear sir , I am having issues running this code on my system , please help me

**Comment by jatinarora1** [comment]
Hi wajid,

Can you please explain what difficulty you are facing while running this on your machine?

**Reply by Wajid4040** [reply]
thanks a lot for replying sir 
I am feeling lucky that you replied me I am software engineering student of 2nd semester and I am found of your project 

I have following questions sir ?
1)How to open this project i.e on which software   ( i am using pycharm and jupyter ) but it is showing error
2) Does we have to open it in jupyter too

thanks a lot

**Comment by Wajid4040** [comment]
Dear sir , 
                   If you have any youtube channel showing how to run this project , then it will help a lot

**Reply by kush27082000** [reply]
@jatinarora1 Please share the guide, How to run this Project
